manager/watcher/raw/raw.go

```diff
@@ -133,10 +133,12 @@ func (self *rawContainerWatcher) watchDirectory(dir string, containerName string
 	}
 	for _, entry := range entries {
 		if entry.IsDir() {
-			// TODO(vmarmol): We don't have to fail here, maybe we can recover and try to get as many registrations as we can.
-			_, err = self.watchDirectory(path.Join(dir, entry.Name()), path.Join(containerName, entry.Name()))
+			entryPath := path.Join(dir, entry.Name())
+			_, err = self.watchDirectory(entryPath, path.Join(containerName, entry.Name()))
```

Comment:
Yes, I agree with @Random-Liu. It doesn't ignore the error on the base directory, but if there is an error on a sub-directory, I think continuing at a best-effort is reasonable.